Update ReadMe repos to @madprime

BenjaminHCCarr opened the pull request:
Change PGP to OH.

BenjaminHCCarr committed:
Update GenNotes repo for ReadMe

Change PGP to OH.

madprime commented:
Actually I think it should be https://github.com/madprime/gennotes again -- I did the same thing, mostly a full reboot, but copied over old notes. (did get revision history though)

BenjaminHCCarr committed:
update repos

changed all refs to `https://github.com/madprime/` I had missed the git call on line #L55

BenjaminHCCarr commented:
I update all repos to `https://github.com/madprime/` this should clear #3.
